src/main/java/edu/ucla/library/avpairtree/verticles/MainVerticle.java

```diff
@@ -130,7 +130,7 @@ private void configureServer(final JsonObject aConfig, final Promise<Void> aProm
                     final List<Future> futures = new ArrayList<>();
 
                     futures.add(deployVerticle(new WatcherVerticle(), aConfig));
-                    futures.add(deployVerticle(new PairtreeVerticle(), aConfig));
+                    futures.add(deployVerticle(new PairtreeVerticle(), aConfig.copy().put(WORKER, true)));
```

Comment:
I'm not actually sure if this helps (or how much), since PairtreeVerticle doesn't really call much blocking code as it is. I figure it can't hurt?

Reply:
Did you compare with and without the pairtree verticle being a worker? The improvement for the converter is great because it's single threaded and has a blocking call but the pairtree verticle isn't (as you say) using any blocking code so giving it its own threads just let's it compete with the converter for resources. The need for making this a worker seems counterintuitive to me since it behaves in an async fashion when it writes files. I'd want to see some numbers showing this is a good improvement.

Reply:
Sounds good. I will revert this change.